src/utils/telemetry.js

```diff
@@ -0,0 +1,200 @@
+import { generateUUID } from './Helper';
+
+let CsTelemetryModule;
+let EkTelemetry;
+let jQuery;
+
+if (typeof window !== 'undefined') {
+  CsTelemetryModule = require('@project-sunbird/client-services/telemetry').CsTelemetryModule;
+  EkTelemetry = require('@project-sunbird/telemetry-sdk');
+  jQuery = require('jquery');
+  window.jQuery = jQuery;
+}
+
+const telemetryConfig = {
+  apislug: '',
+  pdata: {
+    id: 'pratham.portal',
+    pid: '0.0.1',
+    ver: 'pratham.portal'
+  },
+  env: 'pratham.portal',
+  channel: '',
+  did: 'did',
+  authtoken: '',
+  studentid: 'student-id',
```

Comment:
@Aar-if remove studentId, get the userId and track, by default it will be 'Anonymous'